advanced_functionality/pytorch_deploy_large_GPT_model/GPT-J-6B-model-parallel-inference-DJL.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    --endpoint-name gpt-j \
</pre><p>It would be better to use a unique endpoint name in case there are duplicates or this notebook is run multiple times - typically a timestamp is appended to the base name.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3540/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Per @billdoors deferring this change until feature is released.  Difficult to do with CLI implementation; will be updated with boto3 in early sept